neo/Prompt/Utils.py

```diff
@@ -130,6 +133,13 @@ def get_from_addr(params):
     return params, from_addr
 
 
+def get_parse_addresses(params):
+    if '--no-parse' in params:
```

Comment:
just a general thought on the name; should we have these kind of flags be self explanatory? e.g. '--attach-neo' is obvious what it does. '--no-addr-parse' or '--no-addr-convert' might be easier to understand without requiring long help instructions.